build/filter_test.go

```diff
@@ -7,6 +7,8 @@ import (
 	"reflect"
 	"strings"
 
+	"github.com/gophercloud/gophercloud/starlingx/inventory/v1/hosts"
```

Comment:
if gophercloud is mocked, will these imports get removed?

Reply:
Yes dependency will be removed hence the import pkg gets removed